reana-ui/src/pages/launchOnReana/Welcome.module.scss

```diff
@@ -13,6 +13,8 @@
 }
 
 .demo-card {
+  margin-bottom: 2em;
```

Comment:
Now that custom images are possible, we probably need some more CSS tweaks. See as an example the following screenshot:

![](https://github.com/reanahub/reana-ui/assets/13397702/b0392f81-f16e-4f13-8bab-78c8ad649685)

In the second row you can see that the images touch the button below them, plus the third image is also stretched.

As a second example, this is what happens if images are not resized (note that even our default examples change, as the ATLAS RECAST image is not square):

![](https://github.com/reanahub/reana-ui/assets/13397702/a99c3bb8-9a63-4924-a35f-aa7e9f2639a2)

This is instead what happens when images are cropped (note that the ATLAS RECAST plot is cut):

![](https://github.com/reanahub/reana-ui/assets/13397702/f3f995ab-377b-498c-ad09-b05749f3043e)

Possible inspirations/help:
- https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit
- https://react.semantic-ui.com/views/card/
- https://codepen.io/MartijnCuppens/pen/NWNdOBR

Reply:
I would like the image not to be cropped (especially useful when there are graphs shown, as in our case), and at the same time, I would like the images/cards to be all the same height (so that the page looks tidier). I changed a little bit the layout to take advantage of the `Card` component, and took inspiration from the third link you sent to have the background filled by the blurred image.